### ISSUE TITLE
can't find definition TPR

### ISSUE THREAD

**Issue by Kevin-Delnoije** [ifzhang/FairMOT#150]
i am not familiar with the TPR metric of the identity embeddings and i am having a hard time understanding the mathmathical definition and the code that is used for testing the embeddings

```python
    embedding = F.normalize(embedding, dim=1)
    pdist = torch.mm(embedding, embedding.t()).cpu().numpy()
    gt = id_labels.expand(n, n).eq(id_labels.expand(n, n).t()).numpy()

    up_triangle = np.where(np.triu(pdist) - np.eye(n) * pdist != 0)
    pdist = pdist[up_triangle]
    gt = gt[up_triangle]

    far_levels = [1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1]
    far, tar, threshold = metrics.roc_curve(gt, pdist)
    interp = interpolate.interp1d(far, tar)
    tar_at_far = [interp(x) for x in far_levels]
    for f, fa in enumerate(far_levels):
        print('TPR@FAR={:.7f}: {:.4f}'.format(fa, tar_at_far[f]))
```

if you can briefly explain or suggest a resource for understanding this it would be greatly appreciated

**Comment by ifzhang** [comment]
We extract the reid feature using the gt detections and ids. By using the reid feature and its corresponding id, we can evaluate the discriminative ability of the reid features using TPR.  TPR: Apply 1 : N retrieval among these instances and report the true positive rate at false accept rate 0.1 (TPR@FAR=0.1)
